mbedtls/src/rust_printf.c

```diff
@@ -22,7 +28,8 @@ extern int mbedtls_printf(const char *fmt, ...) {
        return -1;
 
     n++;
-    char p[n];
+   //  char p[n];
```

Comment:
Seems this commented code need to be cleaned

Reply:
done in cleanup commit